.golangci.yml

```diff
@@ -17,6 +18,38 @@ linters:
   enable-all: true
   disable:
     # All these break for one reason or another
+    - nolintlint
```

Comment:
I think that overtime we want to actually fix some of those errors instead of suppressing them.

Reply:
I don't not mean `nolintlint` specifically here, mean all suppressed checkers.